src/main/java/com/spotify/docker/client/messages/ContainerInfo.java

```diff
@@ -53,6 +53,7 @@
   @JsonProperty("MountLabel") private String mountLabel;
   @JsonProperty("Volumes") private ImmutableMap<String, String> volumes;
   @JsonProperty("VolumesRW") private ImmutableMap<String, Boolean> volumesRW;
+  @JsonProperty("Node") private Node node;
```

Comment:
I was considering that, but there wasn't a precedent for comments elsewhere in this file ;) Will do.